### DISCUSSION TITLE
Response Cookies

### DISCUSSION THREAD

**Discussion by swassapparel1** [node-fetch/node-fetch#1674] [Q&A]
I'm trying to Post Request on a specific website and wanted to check the response cookies. Tried doing the request via browser and my expected response cookies should be like this.

![expected](https://user-images.githubusercontent.com/79520273/201949743-ee170bb5-9107-4ef5-8fce-f2f32fb89741.JPG)

But when I do post request using node-fetch I'm only getting these cookies

![image](https://user-images.githubusercontent.com/79520273/201949933-b3dd6c89-1cec-4bc1-8cab-38c7ca9e0d80.png)

Please help, thank you!

**Comment by GabenGar** [comment]
Try to include as many headers as possible sent by the browser's request into your request also. Also "trying to Post Request on a specific website and wanted to check the response cookies" sounds like a cross-domain cookie scenario, which would make a lot of servers paranoid, especially if they can guess that request doesn't come from a browser.

**Comment by swassapparel1** [comment]
Hi, I have included all of the request headers same with the browser :(
